core/templates/domain/exploration/exploration-summary.model.ts

```diff
@@ -0,0 +1,82 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Frontend domain model for exploration summary.
+ */
+
+export interface ExplorationSummaryBackendDict {
+  'category': string;
+  'community_owned': boolean;
+  'last_updated_msec': number;
+  'id': string;
+  'created_on_msec': number;
+  'language_code': string;
+  'objective': string;
+  'status': string;
+  'thumbnail_bg_color': string;
+  'thumbnail_icon_url': string;
+  'title': string;
+  'node_count': number;
```

Comment:
Are you sure this is required? Explorations don't have a concept of nodes.

Reply:
> Hi @ChiaraBi, I took a brief look, and it seems like the console error surfaces when you play / preview an exploration (not when you create it). This is the error I see:
> 
> ```ts
> angular.js:15570 TypeError: Cannot convert undefined or null to object
>     at Function.keys (<anonymous>)
>     at eval (exploration-footer.directive.ts:57)
>     at processQueue (angular.js:17948)
>     at angular.js:17996
>     at Scope.$digest (angular.js:19115)
>     at Scope.$apply (angular.js:19503)
>     at done (angular.js:13346)
>     at completeRequest (angular.js:13603)
>     at XMLHttpRequest.requestLoaded [as __zone_symbol__ON_PROPERTYload] (angular.js:13508)
>     at XMLHttpRequest.wrapFn (zone.js:1274) "Possibly unhandled rejection: {}"
> ```
> 
> On taking a closer look at the error, it seems like we are trying to access the `human_readable_contributors_summary` property in the summary model but it doesn't exist -- it should instead be `humanReadableContributorsSummary` as you have defined [here](https://github.com/ChiaraBi/oppia/blob/ab3ceee1a307f70226b34a55f3d2a994fcac53a2/core/templates/domain/exploration/exploration-summary.model.ts#L55).
> 
> https://github.com/ChiaraBi/oppia/blob/ab3ceee1a307f70226b34a55f3d2a994fcac53a2/core/templates/pages/exploration-player-page/layout-directives/exploration-footer.directive.ts#L56-L74
> 

So, should I call it `human_readable_contributors_summary`?

> Can you please take a look at the usages of the api methods (`loadPublicAndPrivateExplorationSummaries` and `loadPublicExplorationSummaries`) to see if similar changes need to be made in other places? Thanks!

I did it, but I was not able to find any useful information about the model's fields, I am sorry.

Reply:
> Are you sure this is required? Explorations don't have a concept of nodes.

Ok, I removed it.